### DISCUSSION TITLE
guide/advanced/offline-download

### DISCUSSION THREAD

**Discussion by giscus** [AlistGo/docs#262] [Announcements]
# guide/advanced/offline-download

Please read the following tips carefully There are two kinds of software with offline download function The two usages are the same, and the software configuration is set in adv...

https://alist.nn.ci/guide/advanced/offline-download.html

**Comment by windyhun** [comment]
我想问下qb服务器是不是需要和alist服务在同一个机器上，因为我路由器上面挂着alist，服务器上挂着qb，用离线下载就会上传失败，显示failed to open file /opt/alist/hc/qbittorrent/3b058078-b251-4755-b48c-213848ade1bb/xxx.zip: open /opt/alist/hc/qbittorrent/3b058078-b251-4755-b48c-213848ade1bb/xxx.zip: no such file or directory
